### DISCUSSION TITLE
Additional CSS properties

### DISCUSSION THREAD

**Discussion by fynnfeldpausch** [taiga-family/taiga-ui#196] [Ideas]
I would like to know or start a discussion about additional CSs properties that you are planning to introduce. From my point of view, introduction of a CSS property becomes more important the harder it is to override them in normal CSS. This especially applies to CSS that is used in mixins, as the result is spread over many places and thus harder to rewrite.

One particular example is the font styles. I think it could be very beneficial to extract font detentions to CSS properties. The property could hold the full font shorthand definition including style, weight, size and line height.

Another example is shadow definitions.

What is your plan and timeframe on this?

**Comment by waterplea** [comment]
Sounds reasonable. Especially considering the mixins are unnecessary once the variables are there. I'll definitely add fonts and box-shadows. If you want this quicker - feel free to open a PR, that would be a welcome addition.

**Comment by waterplea** [comment]
@fynnfeldpausch one thing to keep in mind is — we probably do not need this many fonts varieties. They are required for our proprietary typography guidelines. I think I should just move most of them to proprietary package. Definitely no need to make bold/normal varieties. And line heights also need a closer look — which are used in components and which are not.

**Reply by fynnfeldpausch** [reply]
Thanks for the hints. I will start working on it and make a suggestion within this week.

**Reply by fynnfeldpausch** [reply]
@waterplea Please check #201 and see if you like it :-)
